docs/man_pages/project/testing/debug-ios.md

```diff
@@ -22,6 +22,7 @@ Options:
 * `--debug-brk` - Shorthand for prepare, build and deploy. Prepares, builds and deploys the application package on a device or in an emulator, launches the developer tools of your Safari browser.
 * `--start` - Attaches the debug tools to a deployed and running app. Your app must be running on device or emulator, launches the developer tools of your Safari browser.
 * `--emulator` - Debug on already running emulator. Requires `xcrun` from Xcode 6 or later.
+* `--logging` - If set, prints the standard output of the running application. (Device only)
```

Comment:
At the moment you can't use debugging and logging at the same time on iOS simulator.